docs/data/material/migration/upgrade-to-v7/upgrade-to-v7.md

```diff
@@ -91,11 +84,11 @@ The steps you need to take to migrate from Material UI v6 to v7 are described
 
 The package layout has been updated to use the Node.js exports field. This brings several changes:
 
-Deep imports with more than one level are no longer allowed. For example:
+Deep imports with more than one level are no longer working, at all (they were already considered private API). For example:
 
 ```diff
-- import createTheme from '@mui/material/styles/createTheme';
-+ import { createTheme } from '@mui/material/styles';
+-import createTheme from '@mui/material/styles/createTheme';
++import { createTheme } from '@mui/material/styles';
```

Comment:
Fix the format. One day, we could create a rule: https://github.com/mui/mui-public/issues/249.